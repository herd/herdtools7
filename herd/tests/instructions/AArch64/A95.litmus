AArch64 A95
(* Tests add (shifted register), left shift by 1 *)

{ 0:X0=1; }

P0;
ADD W1, W0, W0, lsl #1;

forall (0:X1=3)

