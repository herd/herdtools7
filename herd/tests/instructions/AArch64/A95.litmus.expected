Test A95 Required
States 1
0:X1=3;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X1=3)
Observation A95 Always 1 0
Hash=993a5ac27bc3548b67b13f7c2abec3b5

